charts/alfresco-transform-service/values.yaml

```diff
@@ -304,32 +344,46 @@ filestore:
     data:
       mountPath: "/tmp/Alfresco"
       subPath: "alfresco-content-services/filestore-data"
-activemq:
-  nameOverride: activemq
-  enabled: false
-  # -- Possibility to choose Node for pod, with a key-value pair label
-  # e.g {"kubernetes.io/hostname": multinode-demo-m02}
-  nodeSelector: {}
-  adminUser:
-    # -- Default username for the embedded broker admin user
-    user: admin
-    # -- Default password for the embedded broker admin user
-    password: admin
-# -- external activemq connection setting when activemq.enabled=false
+  volumes: []
+  volumeMounts: []
+
 messageBroker:
+  # -- External activemq connection url (e.g. failover:(nio://my-broker:61616)?timeout=3000&jms.useCompression=true)
   url: null
+  # -- External activemq username
   user: null
+  # -- External activemq password
   password: null
-  # -- Name of the secret managed by this chart
-  secretName: acs-alfresco-cs-brokersecret
-  # -- Alternatively, provide credentials via an existing secret that contains BROKER_URL, BROKER_USERNAME and BROKER_PASSWORD keys
-  existingSecretName: null
+  existingConfigMap:
+    # -- Alternatively, provide message broker URL via an existing ConfigMap
+    name: null
+    keys:
+      url: BROKER_URL
+  existingSecret:
+    # -- Alternatively, provide message broker credentials via an existing Secret
+    name: null
+    keys:
+      user: BROKER_USERNAME
+      password: BROKER_PASSWORD
+
 ai:
   enabled: false
   nameOverride: alfresco-ai
```

Comment:
to remove

Reply:
I am removing completely the ai chart dependency then